src/gui/qml/credentials/BasicAuthCredentials.qml

```diff
@@ -30,43 +30,51 @@ Credentials {
         text: credentials.isReadOnlyName ? qsTr("Please enter your password to log in.") : qsTr("Please enter %1 and password to log in.").arg(credentials.userNameLabel)
     }
 
-    Label {
-        text: credentials.userNameLabel
-    }
-    TextField {
-        id: userNameField
-        placeholderText: qsTr("Enter %1").arg(credentials.userNameLabel)
-        horizontalAlignment: TextField.AlignHCenter
-        text: credentials.userName
-        enabled: !credentials.isReadOnlyName
-        onTextChanged: {
-            credentials.userName = text;
-        }
+    Pane {
+        // Treat the user name and password fields (and labels) as one single group, so they are
+        // always placed and layed out together.
+        Layout.alignment: Qt.AlignHCenter
 
-        Keys.onBacktabPressed: {
-            widget.parentFocusWidget.focusPrevious();
-        }
-    }
+        ColumnLayout {
+            Label {
+                text: credentials.userNameLabel
+            }
+            TextField {
+                id: userNameField
```

Comment:
I think now that we have this internal Panel, with this new layout, you should set fillWidth for the contained elements.
Also I'm pretty sure you should add the buttons, else weird stuff will happen.

Reply:
done